exporter/chronicleexporter/exporter.go

```diff
@@ -142,3 +175,34 @@ func (ce *chronicleExporter) buildOptions() []grpc.CallOption {
 
 	return opts
 }
+
+func (ce *chronicleExporter) startHostMetricsCollection(ctx context.Context) {
+	ticker := time.NewTicker(5 * time.Minute)
+	defer ticker.Stop()
+
+	ce.wg.Add(1)
```

Comment:
We should increment the wait group outside of the goroutine. There's a race condition where it may not get incremented before shutdown if caused.

Reply:
moved